tutorials/6 Analysis/S Param Simulation.ipynb

```diff
@@ -311,8 +318,8 @@
     "First parameter: List of components to render (empty list if rendering whole Metal design) <br>\n",
```

Comment:
Above, regarding cell for open_ansys(), suggest alternate text.

Skip the next cell if Ansys is already open and with a project loaded.

in bold:
**Wait for Ansys to open up completely before proceeding to the next cell.**